compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -29,7 +29,7 @@ namespace luci
 namespace
 {
 
-void compute_asym_scale_zp(float min, float max, float *scaling_factor, int64_t *zp)
+void compute_asym_scale_zp(float min, float max, float *scaling_factor, int64_t *zp, float *nudged_min, float *nudged_max)
 {
   assert(min != max);
   if (min == max)
```

Comment:
This line seems to not be able to run because of above assert section. Is it intended for debug or sth?